<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GLSpectrumGUI</class>
 <widget class="QWidget" name="GLSpectrumGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>112</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Spectrum</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="Line1Layout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QComboBox" name="fftWindow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>FFT window function</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>Bart</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B-H</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ham</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Han</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kai</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fftSize">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>FFT size</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>256</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>512</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32k</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fftOverlap">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>FFT overlap</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>63</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="averagingMode">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Averaging type</string>
       </property>
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mov</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Max</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="averaging">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Number of averaging samples</string>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1e5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2e5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5e5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1M</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="freeze">
       <property name="toolTip">
        <string>Play/Pause spectrum display</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/pause.png</normaloff>
         <normalon>:/play.png</normalon>:/pause.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Line2Layout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="autoscale">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Autoscale max level and range</string>
       </property>
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="refLevel">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Reference level (dB)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>-110</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="levelRange">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Range (dB)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fps">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Spectrum maximum FPS (NL for No Limit)</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="linscale">
       <property name="toolTip">
        <string>Logarithmic / Linear scale selection</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/logarithmic.png</normaloff>
         <normalon>:/linear.png</normalon>:/logarithmic.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="wsSpectrum">
       <property name="toolTip">
        <string>Left: toggle websocket spectrum - Right: websocket spectrum parameters</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/stream.png</normalon>
        </iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="markers">
       <property name="toolTip">
        <string>Open spectrum markers dialog</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/gridpolar.png</normalon>
        </iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Line3Layout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="clearSpectrum">
       <property name="toolTip">
        <string>Clear spectrum histogram</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="histogram">
       <property name="toolTip">
        <string>Display phosphor effect spectrum</string>
       </property>
       <property name="text">
        <string>Histogram</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/histogram.png</normaloff>:/histogram.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="decay">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Decay:</string>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="decayDivisor">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Decay divisor:</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="stroke">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Stroke:</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="maxHold">
       <property name="toolTip">
        <string>Display max hold</string>
       </property>
       <property name="text">
        <string>Max Hold</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/maxhold.png</normaloff>:/maxhold.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="current">
       <property name="toolTip">
        <string>Display live spectrum</string>
       </property>
       <property name="text">
        <string>Max Hold</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/current.png</normaloff>:/current.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="traceIntensity">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Trace intensity</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="invertWaterfall">
       <property name="toolTip">
        <string>Exchange waterfall and histogram</string>
       </property>
       <property name="text">
        <string>Inv</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/invertspectrum.png</normaloff>:/invertspectrum.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="waterfall">
       <property name="toolTip">
        <string>Display waterfall</string>
       </property>
       <property name="text">
        <string>Waterfall</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/waterfall.png</normaloff>:/waterfall.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="grid">
       <property name="toolTip">
        <string>Toggle the scale grid</string>
       </property>
       <property name="text">
        <string>Grid</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/grid.png</normaloff>:/grid.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="gridIntensity">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Grid intensity</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>maxHold</tabstop>
  <tabstop>invertWaterfall</tabstop>
  <tabstop>grid</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
